packages/larva-css/src/02-algorithms/a-flex/a-flex.common.inline.scss

```diff
@@ -0,0 +1,30 @@
+@import 'setup';
```

Comment:
One more change - this should be a utility since it is a single declaration that is expressed in the class name. Can you please move this to that directory?

Also, when generating sets of utilities with breakpoints, all classes must be grouped by breakpoint so that they do not override each other. See this file for an example: https://github.com/penske-media-corp/pmc-larva/blob/master/packages/larva-css/src/03-utilities/u-padding.common.inline.scss